resources/serverless/values.yaml

```diff
@@ -4,6 +4,12 @@
 
 fullnameOverride: "serverless"
 
+bindingsMigration:
+  image:
+    repository: 'dtzar/helm-kubectl'
```

Comment:
Most of the images I was looking for have only kubectl, and this has also apk, shell etc. what is needed in migration job.

Reply:
Have a look at https://github.com/kyma-project/kyma/blob/master/resources/rafter/values.yaml#L331

it has kubectl, helm, jq etc, I think everything that you need, and it's "blessed" by our testinfra team -> no security issues with it